src/OBO.NET/OboOntology.fs

```diff
@@ -519,6 +519,47 @@ type OboOntology =
     static member getSynonyms term (onto : OboOntology) =
         onto.GetSynonyms term
 
+    /// <summary>Checks if the given terms are treated as equivalent in the OboOntology.</summary>
+    /// <remarks>Note that term1 must be part of the OboOntology while term2 must be part of another ontology.</remarks>

```

Comment:
I think this is fine for now, but i think we should eventually either add a check here or add item accession to the terms in the ontology, e.g. ontology["termName"] should return the relevant term, so this method just needs to be `this.AreTermsEquivalent(termName : string, externalTerm : OboTerm) = ...`.

Requiring one term being from the ontology without checking/enforcing it screams for unintended behavior IMO, but as said, fine for now